<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenSubtitlesSearch</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/opensubtitles</normaloff>:/opensubtitles</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="SubtitlesView" name="subtitlesTableView">
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::NoDragDrop</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::IgnoreAction</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionConnexion"/>
    <addaction name="actionD_connexion"/>
    <addaction name="separator"/>
    <addaction name="actionSearch"/>
    <addaction name="separator"/>
    <addaction name="actionPropri_t_s"/>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuSous_titres">
    <property name="title">
     <string>Subtitles</string>
    </property>
    <addaction name="actionD_tail"/>
    <addaction name="actionT_l_charger"/>
    <addaction name="actionComment_subtitle"/>
    <addaction name="actionAssocier_une_vid_o"/>
    <addaction name="separator"/>
    <addaction name="actionLien_vers_page_du_sous_titre"/>
    <addaction name="actionLien_t_l_chargement_zip"/>
    <addaction name="actionLien_t_l_chargement_gzip"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionA_propos"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuSous_titres"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Toolbar</string>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPropri_t_s"/>
   <addaction name="separator"/>
   <addaction name="actionSearch"/>
   <addaction name="separator"/>
   <addaction name="actionD_tail"/>
   <addaction name="actionT_l_charger"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="toolTip">
    <string>Transaction status</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="accessibleDescription">
    <string/>
   </property>
   <property name="windowTitle">
    <string>Network view</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="NetworkReplyDownloadView" name="networkReplytableView">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetDetail">
   <property name="windowTitle">
    <string>Detail view</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="SubtitleDetail" name="subtitleDetail">
       <property name="undoRedoEnabled">
        <bool>false</bool>
       </property>
       <property name="lineWrapMode">
        <enum>QPlainTextEdit::NoWrap</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionPropri_t_s">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/settings</normaloff>:/settings</iconset>
   </property>
   <property name="text">
    <string>Properties</string>
   </property>
   <property name="toolTip">
    <string>Modify general properties</string>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/ico/exit.png</normaloff>:/ico/exit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit the application</string>
   </property>
  </action>
  <action name="actionT_l_charger">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/download</normaloff>:/download</iconset>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionConnexion">
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect to OpenSubtitles server</string>
   </property>
  </action>
  <action name="actionD_connexion">
   <property name="text">
    <string>Disconnect</string>
   </property>
   <property name="toolTip">
    <string>Disconnect from OpenSubtitles server</string>
   </property>
  </action>
  <action name="actionD_tail">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/voir</normaloff>:/voir</iconset>
   </property>
   <property name="text">
    <string>Detail</string>
   </property>
   <property name="toolTip">
    <string>Show subtitle detail view</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionLien_t_l_chargement_gzip">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/lien</normaloff>:/lien</iconset>
   </property>
   <property name="text">
    <string>Download link (gzip)</string>
   </property>
   <property name="toolTip">
    <string>Download link (gzip)</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionLien_t_l_chargement_zip">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/lien</normaloff>:/lien</iconset>
   </property>
   <property name="text">
    <string>Download link (zip)</string>
   </property>
   <property name="toolTip">
    <string>Download link (zip)</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionLien_vers_page_du_sous_titre">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/lien</normaloff>:/lien</iconset>
   </property>
   <property name="text">
    <string>Subtitle link</string>
   </property>
   <property name="toolTip">
    <string>Open subtitle web page</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionA_propos">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/apropos</normaloff>:/apropos</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About OpenSubtitlesSearch</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAssocier_une_vid_o">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/lierVideo</normaloff>:/lierVideo</iconset>
   </property>
   <property name="text">
    <string>Associate a video hash</string>
   </property>
   <property name="toolTip">
    <string>Associate video hash with selected subtitles</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSearch">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/recherche</normaloff>:/recherche</iconset>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
   <property name="toolTip">
    <string>Search subtitles</string>
   </property>
  </action>
  <action name="actionComment_subtitle">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/comment</normaloff>:/comment</iconset>
   </property>
   <property name="text">
    <string>Comment subtitle</string>
   </property>
   <property name="toolTip">
    <string>Comment subtitle</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SubtitlesView</class>
   <extends>QTableView</extends>
   <header>subtitlesview.h</header>
   <slots>
    <signal>hasSelection(bool)</signal>
    <signal>hasSingleSelection(bool)</signal>
    <slot>openDownloadGzLink()</slot>
    <slot>openDownloadZipLink()</slot>
    <slot>openShowLink()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>NetworkReplyDownloadView</class>
   <extends>QTableView</extends>
   <header>networkreplydownloadview.h</header>
  </customwidget>
  <customwidget>
   <class>SubtitleDetail</class>
   <extends>QPlainTextEdit</extends>
   <header>subtitledetail.h</header>
   <slots>
    <slot>subtitlesListSelectionChanged()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionPropri_t_s</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>editProperties()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuitter</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionT_l_charger</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>rawDownloadCurrentSubtitle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionD_tail</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showDetail()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>subtitlesTableView</sender>
   <signal>hasSelection(bool)</signal>
   <receiver>actionT_l_charger</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLien_t_l_chargement_gzip</sender>
   <signal>triggered()</signal>
   <receiver>subtitlesTableView</receiver>
   <slot>openDownloadGzLink()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLien_t_l_chargement_zip</sender>
   <signal>triggered()</signal>
   <receiver>subtitlesTableView</receiver>
   <slot>openDownloadZipLink()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLien_vers_page_du_sous_titre</sender>
   <signal>triggered()</signal>
   <receiver>subtitlesTableView</receiver>
   <slot>openShowLink()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>subtitlesTableView</sender>
   <signal>hasSingleSelection(bool)</signal>
   <receiver>actionLien_t_l_chargement_gzip</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>subtitlesTableView</sender>
   <signal>hasSingleSelection(bool)</signal>
   <receiver>actionLien_t_l_chargement_zip</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>subtitlesTableView</sender>
   <signal>hasSingleSelection(bool)</signal>
   <receiver>actionLien_vers_page_du_sous_titre</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionA_propos</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showAPropos()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>subtitlesTableView</sender>
   <signal>hasSelection(bool)</signal>
   <receiver>actionAssocier_une_vid_o</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAssocier_une_vid_o</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showUpdateMovieHash()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSearch</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showSearchDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionComment_subtitle</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showAddComment()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>subtitlesTableView</sender>
   <signal>hasSingleSelection(bool)</signal>
   <receiver>actionComment_subtitle</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>editProperties()</slot>
  <slot>showSearchByQuery()</slot>
  <slot>rawDownloadCurrentSubtitle()</slot>
  <slot>showSearchByHash()</slot>
  <slot>showDetail()</slot>
  <slot>showAPropos()</slot>
  <slot>showUpdateMovieHash()</slot>
  <slot>showSearchDialog()</slot>
  <slot>showAddComment()</slot>
 </slots>
</ui>
